recipe/meta.yaml

```diff
@@ -31,6 +31,32 @@ requirements:
     - crosstool-ng >=1.24.0,<1.25
 
 outputs:
+  - name: libgcc-devel_{{ ctng_target_platform }}
+    script: install-libgcc-devel.sh
+    build:
+      number: {{ build_num }}
+      detect_binary_files_with_prefix: False
+      binary_relocation: False
```

Comment:
Why is all binary relocation off here?

Reply:
Because that's what is used for others. The compilers are relocatable, so they don't need their prefixes changed. If there is one accidentally, we should ignore it. Otherwise, doing `make install` in outputs will trigger recompilation as the binary would change. (If `binary_relocation` is turned on and there's a prefix in the binary, `conda` will patch it for each output in a recipe and that will trigger `make` to a full rebuild. I'll try and dig up the issue.